deployment/ccip/changeset/crossfamily/v1_6/cs_add_evm_solana_lane.go

```diff
@@ -0,0 +1,442 @@
+package v1_6
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/chainlink-ccip/chains/evm/gobindings/generated/v1_6_0/fee_quoter"
+	cldf "github.com/smartcontractkit/chainlink-deployments-framework/deployment"
+	"github.com/smartcontractkit/chainlink-deployments-framework/operations"
+	mcmslib "github.com/smartcontractkit/mcms"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	ccipchangeset "github.com/smartcontractkit/chainlink/deployment/ccip/changeset"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/changeset/solana"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/changeset/v1_6"
+	commonstate "github.com/smartcontractkit/chainlink/deployment/common/changeset/state"
+	"github.com/smartcontractkit/chainlink/deployment/common/proposalutils"
+)
+
+var (
+	AddEVMAndSolanaLaneChangeset = cldf.CreateChangeSet(addEVMAndSolanaLaneLogic, addEVMSolanaPreconditions)
+
+	postOps = operations.NewOperation(
+		"postOpsToAggregateProposals",
+		semver.MustParse("1.0.0"),
+		"Post ops to aggregate proposals",
+		func(b operations.Bundle, deps Dependencies, input postOpsInput) ([]mcmslib.TimelockProposal, error) {
+			allProposals := input.Proposals
+			proposal, err := proposalutils.AggregateProposals(
+				deps.Env, deps.EVMMCMSState, deps.SolanaMCMSState, allProposals, nil,
+				"Adding EVM and Solana lane", input.MCMSConfig)
+			if err != nil {
+				return nil, err
+			}
+			if proposal != nil {
+				input.Proposals = []mcmslib.TimelockProposal{*proposal}
+			}
+			return input.Proposals, nil
+		},
+	)
+
+	addEVMAndSolanaLaneSequence = operations.NewSequence(
+		"addEVMAndSolanaLane",
+		semver.MustParse("1.0.0"),
+		"Adds bi-directional lane between EVM chain and Solana",
+		func(b operations.Bundle, deps Dependencies, input AddRemoteChainE2EConfig) ([]mcmslib.TimelockProposal, error) {
+			deps.Env.Logger.Infow("Adding EVM and Solana lane", "EVMChainSelector", input.EVMChainSelector, "SolanaChainSelector", input.SolanaChainSelector)
+			var finalOutput deployment.ChangesetOutput
```

Comment:
These doesn't seem to be a need to initialize a `ChangesetOutput` here. Instead it would be better to only declare exactly what you need (`[]MCMSTimelockProposals`)